python/pdstools/reports/ModelReport.qmd

```diff
@@ -16,26 +16,35 @@ format:
     toc: true
     toc-title: Table of Contents
     theme:
-        light: flatly
-        dark: darkly
+      light: flatly
+      dark: darkly
 jupyter: python3
 ---
 
 ```{python}
+import sys
+sys.path.append('/Users/perdo/Documents/pega/pega-datascientist-tools/python')
+
 from pdstools import datasets, ADMDatamart
+from pdstools.utils.cdh_utils import zRatio, lift
 import polars as pl
 from IPython.display import display, Markdown
+from tabulate import tabulate
+from itables import show
 import os.path
+import plotly.express as px
 ```
 
 ```{python}
 #| tags: [parameters]
 
 # These parameters are overwritten when called externally
+
+# TODO see if we can align with the params of the Health Check
+
 datafolder = os.path.expanduser("~/Downloads/tmp/")
```

Comment:
Could you link this to a url location maybe?

Reply:
?

Reply:
I meant rather than having a hardcoded url to /downloads/tmp, are the files in github? Then you can just link to the file in github directly